Holo-Host/hp-admin
# D-01075 - Refetch updates

### Updates
 - decreases polling interval to 60000ms
 - adds refresh button in Side Navigation Bar

## To Test : 
 - spin up a single agent with sim2h server
 - once in holofuel, open up the side nav and press the refresh button
  - notice the network call logs in the console and updated data appear on page, if available/relevant
